tests/bundlers/test_venv_bundler.py

```diff
@@ -268,3 +268,33 @@ def test_bundler_can_filter_dependency_groups(
   • Bundled simple-project (1.2.3) into {path}
 """
     assert expected == io.fetch_output()
+
+
+@pytest.mark.parametrize("compile", [True, False])
+def test_bundler_passes_compile_flag(
```

Comment:
How does this test check that the compile flag is passed correctly to the installer? If I did not miss anything there is no difference between passing the flag and not passing the flag.

Reply:
yeah, my bad
I pushed a new commit that mocks the executor method to check if flag is passed